src/cpp/fastdds/subscriber/DataReaderImpl/ReadTakeCommand.hpp

```diff
@@ -65,7 +65,8 @@ struct ReadTakeCommand
             int32_t max_samples,
             const StateFilter& states,
             const history_type::instance_info& instance,
-            bool single_instance = false)
+            bool single_instance = false,
+            bool loop_for_data = false)
```

Comment:
I would take the opportunity to remove the default values here. Being explicit is the safest way to avoid accidental mistakes.

Reply:
done